### ISSUE TITLE
Improve error handling for wlan

### ISSUE THREAD

**Issue by vpetersson** [NebraLtd/hm-diag#83]
It looks like the diagnostics tool is running just fine, but we're getting this error:

```
 diagnostics  ERROR:root:[Errno 2] No such file or directory: '/sys/class/net/wlan0/address'
 diagnostics  grep: /sys/bus/usb/devices/*/idVendor: No such file or directory
 diagnostics  grep: /sys/bus/usb/devices/*/idVendor: No such file or directory
```

We need to improve the error handling for this.

**Comment by KevinWassermann94** [comment]
Is this the error that is causing the Wifi issues on some units?

**Comment by vpetersson** [comment]
No, this code that caused this was only briefly deployed on testnet. It has not been deployed at all to mainnet.

**Comment by shawaj** [comment]
It is a related problem though. I'm pretty sure it's a WiFi driver issue for the rtl8188 chipset

**Comment by marvinmarnold** [comment]
I get this error on [master](https://github.com/NebraLtd/hm-diag/commit/5436da7e8a06cae35791cf00381abe77ed4885f2) when I run on my local machine or deploying to a Raspberry Pi 3 B+. PR #92 appears to resolve this issue. [This](https://github.com/NebraLtd/hm-diag/pull/92/files#diff-1bc7190be6e1a8de2f5a4d659238a140f25dfff6cffa3587bc2e317b6d430709R15) is the relevant code but I haven't looked to see how it is different than master. @robputt?

**Comment by vpetersson** [comment]
@marvinmarnold Basically master isn't useful atm. We did some attempts to refactor it prior to @robputt beginning the work. What is "working" (i.e. running on production hotspots) is 8d4c300237e559e2c1b6e873a4f5b09f78cdfdad. I hope that we will be able to deploy @robputt major refactoring work shortly.
